datalad/support/archives.py

```diff
@@ -81,7 +86,19 @@ def _patool_run(cmd, verbosity=0, **kwargs):
     # use our runner
     try:
         # kwargs_ = kwargs[:];         kwargs_['shell'] = True
-        _runner.run(cmd, **kwargs)
+        # Any debug/progress output could be spit out to stderr so let's
+        # "expect" it.
+        #
+        if isinstance(cmd, (list, tuple)) and kwargs.get('shell'):
+            # patool (as far as I see it) takes care about quoting args
+            cmd = ' '.join(cmd)
```

Comment:
Also we might want to consider moving this into the runner itself. I didn't spot any place where we use shell and pass the list, so it shouldn't change any other behavior but might prevent us from hitting it again